README.md

```diff
@@ -7,7 +7,7 @@ Generic TypeScript Merger
 
 ```javascript
 let tsm = require('@devonfw/ts-merger');
-let mergedCode: string = tsm.merge(baseContents, pathContents, patchOverrides);
```

Comment:
Why does Github show that there were Changes here?

Reply:
I have corrected the spelling from "pathContents" (missing c), to "patchContents"

Reply:
Now I see it too I have not noticed it at all 🤦